python/ray/tune/search/nevergrad/nevergrad_search.py

```diff
@@ -120,6 +122,7 @@ def __init__(
         optimizer: Optional[
             Union[Optimizer, Type[Optimizer], ConfiguredOptimizer]
         ] = None,
+        optimizer_kwargs: Dict = dict(),
```

Comment:
We can avoid the mutable default object here.

```suggestion
        optimizer_kwargs: Optional[Dict] = None,
```